jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/actions/OpenCurrentInEditor.kt

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudwatch.logs.actions
+
+import com.intellij.icons.AllIcons
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.project.DumbAware
+import com.intellij.openapi.project.Project
+import com.intellij.util.ui.ListTableModel
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+import software.amazon.awssdk.services.cloudwatchlogs.model.OutputLogEvent
+import software.aws.toolkits.jetbrains.utils.ApplicationThreadPoolScope
+import software.aws.toolkits.jetbrains.utils.getCoroutineUiContext
+import software.aws.toolkits.resources.message
+
+class OpenCurrentInEditor(
```

Comment:
Is the point of this to open the current log state in an editor? Do we want to handle log streaming in an editor?

Reply:
Yes, but no we do not handle streaming. It just opens the current view, which makes more sense once we add search/filter